db/migrate/20131002004641_create_automatic_exploitation_matches.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAutomaticExploitationMatches < ActiveRecord::Migration
+  def change
+    create_table :automatic_exploitation_matches do |t|
+      t.integer :ref_id
+      t.string :state
+      t.integer :nexpose_data_vulnerability_definition_id
+
+      t.timestamps
+    end
+
+    add_index :automatic_exploitation_matches, :ref_id
```

Comment:
Why no index on `:nexpose_data_vulnerability_definition_id`?  All foreign keys should be indexed.